components/enedis/enedis.app.mjs

```diff
@@ -0,0 +1,123 @@
+import { axios } from "@pipedream/platform";
+
+export default {
+  type: "app",
+  app: "enedis",
+  propDefinitions: {
+    start: {
+      label: "Start",
+      description: "Start date of the period in RFC-3339 format (YYYY-MM-DD) e.g. '2022-10-28'. The returned datas will include the start date.",
+      type: "string",
+    },
+    end: {
+      label: "End",
+      description: "End date of the period in RFC-3339 format (YYYY-MM-DD) e.g. '2022-11-01'. The returned datas will not include the end date.",
+      type: "string",
+    },
+    usage_point_id: {
+      label: "Usage point id",
+      description: "Usage point id known by the user. Use '12655648759651' if don't have one.",
```

Comment:
Does it make sense to have this default value available, or is this just for testing?

Reply:
Yes it makes sence. The usage point id refer to electricity delivery points in France. I have my own usage point which I can use in sandbox mode and production mode. I can send it to you by via slack if you want.

As a US tester, you don't have your own usage point id. You can use fictif ones (like `12655648759651`) or [any others defined here](https://datahub-enedis.fr/data-connect/ressources/decouvrir/) in sandbox mode only.